_run/common-commands.mk

```diff
@@ -55,6 +59,18 @@ provider-update:
 provider-status:
 	$(AKASH) provider status $(PROVIDER_ADDRESS)
 
+.PHONY: jwt-server-authenticate
+jwt-server-authenticate:
+	$(AKASH) provider jwt-server-authenticate \
```

Comment:
let's just call this command `authenticate` - `akash provider authenticate`.

Reply:
Done